slides/_preparation.qmd

```diff
@@ -23,13 +23,13 @@ pip install ./
 \
 
 :::{.fragment}
-Navigate to `exercises/exercise_00/` where you will see a python and
-fortran code.
+Navigate to `exercises/exercise_00/` where you will see both Python and
+Fortran scripts.
```

Comment:
Do we 'script' Fortran?

Reply:
Addressed in 9c9551aff5da24f1d3b47593d7504a0b2a81eb04.